src/app/services/log.service.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { TestBed } from '@angular/core/testing';
+
+import { LogService } from './log.service';
+
+describe('LogService', () => {
+  let service: LogService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(LogService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should output the given message as an error to the console', () => {
+    const error = jest.spyOn(console, 'error').mockImplementation();
+    const errorMessage = "This is an error message";
+    service.error(errorMessage);
+    expect(error).toHaveBeenCalled();
+    expect(error).toBeCalledWith('%c' + errorMessage, 'color: #DC143C');
```

Comment:
With this `expect(...).toBeCalledWith(...)`, the first `expect(...)` is not necessary. The second one will ensure that (a) the method was called and (b) it was called with the correct arguments. It does double-duty